deploy_and_install_scylla_ansible_script/scylla_deployment.yml

```diff
@@ -0,0 +1,170 @@
+#     Update apt cache on Debian / Ubuntu                                                       TAGS: [1_prereq]
+#     Install add-apt-repository command utility on Debian / Ubuntu14                           TAGS: [1_prereq]
+#     Add openjdk PPA to Ubuntu14 (prereq for Java 8)                                           TAGS: [2_java]
+#     Add openjdk PPA to Debian (prereq for Java 8)                                             TAGS: [2_java]
+#     Add Jessie-backports repo to Debian (prereq for Java 8)                                   TAGS: [2_java]
+#     Update apt cache on Debian / Ubuntu                                                       TAGS: [2_java]
+#     Install Java 8 on Ubuntu14, needed for Scylla release 1.7                                 TAGS: [2_java]
+#     Install Java 8 on Debian, needed for Scylla release 1.7                                   TAGS: [2_java]
+#     Select correct java version on Debian / Ubuntu14                                          TAGS: [2_java]
+#     Download Scylla {{ release }} repo for Centos 7 / RHEL 7                                  TAGS: [3_repo]
+#     Download Scylla {{ release }} repo for Ubuntu 14.04 (Trusty)                              TAGS: [3_repo]
+#     Download Scylla {{ release }} repo for Ubuntu 16.04 (Xenial)                              TAGS: [3_repo]
+#     Download Scylla {{ release }} repo for Debian 8 (Jessie)                                  TAGS: [3_repo]
+#     Install scylla {{ release }} on Debian / Ubuntu                                           TAGS: [4_install]
+#     Install scylla {{ release }} on CentOS / RHEL                                             TAGS: [4_install]
+#     Configure Cluster name in yaml file                                                       TAGS: [5_conf]
+#     Configure seeds in yaml file                                                              TAGS: [5_conf]
+#     Configure listen address + rpc address in yaml file                                       TAGS: [5_conf]
+#     Run Scylla Setup (RAID-0, XFS format, NIC queue, disk IOtune), this may take a while      TAGS: [5_conf]
+#     Reboot server/s (required by Scylla)                                                      TAGS: [6_reboot]
+#     Wait for server/s to come up from boot                                                    TAGS: [6_reboot]
+
+
+# List all facts collected by ansible, on the host group: ansible -i servers.ini servers -m setup | less
+# To filter specific facts use: -a "filter=ansible_distribution*"
+
+
+---
+
+- hosts: [host_group_name_from_ini_file]
+  become: yes                                   # Run all tasks as root
+  vars:
+    release: 1.7                                # Scylla release to be installed
+    cluster_name: cluster_name                  # Unique cluster name
+    seeds: seed_node_ip                         # Need at least 1 live seed node for new nodes to join the cluster (use 1:3 ratio)
```

Comment:
All these vars, the user need to edit and add them - for example:
- release: 1.7
- cluster_name: cluster_name_example
- seeds: ip of to-be seed node (ratio of 1:3, as mentioned in the comment)
- disks: /dev/sdb,/dev/sdc
- NIC: eth0

I'll add it to the README.md file